Vagrantfile

```diff
@@ -1,14 +1,13 @@
 # vim:ft=ruby
 
 Vagrant.configure("2") do |config|
- 	config.vm.box_url = 'http://files.vagrantup.com/precise32.box'
-	config.vm.box = "precise32"
+	config.vm.box = "ubuntu/trusty64"
 	config.vm.host_name = "portia"
-	config.vm.provision :shell, :path => 'provision.sh'
+	config.vm.provision :shell, :path => 'provision.sh', :args => ["install_deps", "install_splash", "install_python_deps", "configure_nginx", "configure_initctl"]
 	config.vm.network "private_network", ip: "33.33.33.10"
-	config.vm.network "forwarded_port", guest: 8000, host: 8000
+	config.vm.network "forwarded_port", guest: 9001, host: 9001
```

Comment:
Is there any reason to change this? People who currently use this already have their links set for port 8000 and might be confused when it stops working.